client/src/app/pages/issues/helpers.ts

```diff
@@ -195,3 +200,10 @@ export const parseReportLabels = (
   });
   return { sources, targets, otherLabels };
 };
+
+export const getIssueTitle = (
+  issueReport: AnalysisRuleReport | AnalysisIssue | AnalysisIssueReport
+) =>
+  issueReport?.description?.split("\n")[0] ??
+  issueReport?.name?.split("\n")[0] ??
+  "*Unnamed*";
```

Comment:
Is empty string a valid issue title?

One of the main differences between `||` and `??` is how empty string (and 0) is handled.  For `??` they're considered valid values, and for `||` they're considered `false`.  Here with `??`, if the `description` is empty string, the title will be empty string.

`"".split("\n")[0] ?? "foo"` is `""`

`"".split("\n")[0] || "foo"` is `"foo"`

Reply:
Good to know! I think I will revert back to `||`. We don't wan't an empty string title.